example/builder/src/new_class_consumed_props.dart

```diff
@@ -32,13 +32,14 @@ class SomeParentProps = UiProps with ParentOnlyPropsMixin, SharedPropsMixin;
 class SomeClassParentComponent extends UiComponent2<SomeParentProps> {
   @override
   render() {
-    final meta = UiPropsMeta(props).meta;
-    meta.consume(ParentOnlyPropsMixin);
+    final meta = UiPropsMeta(props).meta.forMixin(ParentOnlyPropsMixin);
```

Comment:
#nit usage of the UiPropsMeta wrapper isn't necessary in cases where there isn't already a `meta field`
```suggestion
    final meta = props.meta.forMixin(ParentOnlyPropsMixin);
```
I think we should omit this where possible

Reply:
Unfortunately this always seems necessary :/ Without it, I get `error: Static field 'meta' can't be accessed through an instance. (instance_access_to_static_member at [over_react] example/builder/src/new_class_consumed_props.dart:35)`

Reply:
I could rename it to `propsMeta`, `instanceMeta`, or something to that effect though. I can't think of anything really succinct, but those could still be better than requiring always wrapping

Reply:
Oh gross, because it's conflicting with the generated static `meta` fields on the legacy props mixins mixed into UiProps itself...

I knew that would be an issue for legacy props classes annd mixins, but I didn't realize that those were being mixed into `UiProps`...

For example, removing these fixes it: 
<details>
<summary> <em>Click to expand</em> </summary>

```diff
diff --git a/example/builder/src/new_class_consumed_props.dart b/example/builder/src/new_class_consumed_props.dart
index c2f9a972..55f9b4c8 100644
--- a/example/builder/src/new_class_consumed_props.dart
+++ b/example/builder/src/new_class_consumed_props.dart
@@ -32,7 +32,7 @@ class SomeParentProps = UiProps with ParentOnlyPropsMixin, SharedPropsMixin;
 class SomeClassParentComponent extends UiComponent2<SomeParentProps> {
   @override
   render() {
-    final meta = UiPropsMeta(props).meta.forMixin(ParentOnlyPropsMixin);
+    final meta = props.meta.forMixin(ParentOnlyPropsMixin);
 
     return (
         Dom.div()(
@@ -58,4 +58,4 @@ class SomeClassChildComponent extends UiComponent2<SomeChildProps> {
         )
     );
   }
-}
\ No newline at end of file
+}
diff --git a/lib/src/component/prop_mixins.over_react.g.dart b/lib/src/component/prop_mixins.over_react.g.dart
index 2141f846..d980d03d 100644
--- a/lib/src/component/prop_mixins.over_react.g.dart
+++ b/lib/src/component/prop_mixins.over_react.g.dart
@@ -11,7 +11,6 @@ abstract class ReactPropsMixin implements _$ReactPropsMixin {
   @override
   Map get props;
 
-  static const PropsMeta meta = _$metaForReactPropsMixin;
   @override
   String get key => props['key'];
   @override
@@ -6765,7 +6764,6 @@ abstract class UbiquitousDomPropsMixin implements _$UbiquitousDomPropsMixin {
   @override
   Map get props;
 
-  static const PropsMeta meta = _$metaForUbiquitousDomPropsMixin;
   @override
   @Accessor(doNotGenerate: true)
   AriaPropsMapView _aria;
diff --git a/lib/src/util/class_names.over_react.g.dart b/lib/src/util/class_names.over_react.g.dart
index 391c9f00..d7a7a80f 100644
--- a/lib/src/util/class_names.over_react.g.dart
+++ b/lib/src/util/class_names.over_react.g.dart
@@ -11,8 +11,6 @@ abstract class CssClassPropsMixin implements _$CssClassPropsMixin {
   @override
   Map get props;
 
-  static const PropsMeta meta = _$metaForCssClassPropsMixin;
-
   /// String of space-delimited CSS classes to be added to the resultant DOM.
   ///
   /// All over_react components merge any added classes with this prop and the [classNameBlacklist] prop (see
```

</details>

Ugh we really need to update those to be non-legacy mixins at some point... I'm gonna add that to the list for the next breaking change.

Anyways, yeah, I'm in favor of renaming it so that you don't have to wrap it. It's probably a good thing that this happened since the name `meta` was pretty ambiguous to begin with...

I think it'd be nice if the name made it clear that it's specific to the type/class and not the instance itself, so my vote would be for something like `propsMeta`/`staticMeta`/`classMeta`. At some point, I think it'd be good to get some input from the rest of the squad/team, and maybe even from consumers, to make sure the name is a good fit.

Reply:
Sweeet, yeah, I agree it could be more descriptive and better to just rename it. I made a poll in slack for our squad (and anyone else in the channel). Based off of that, I'll rename it!